app/views/questions/upload-supporting-documentation.html

```diff
@@ -7,26 +7,50 @@
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
       <div class="govuk-caption-m">{{ title }}</div>
-      <h1 class='govuk-heading-l'>Upload supporting documents</h1>
+      <h1 class='govuk-heading-l'>
+        Upload supporting documents
+       </h1>
       <p class="govuk-body">
-        Upload supporting documentation for your {{ title }} form below.
-      </p>
+        Upload extra information or evidence in support of your {{ title }} application.
+       </p>
+
       <div class="govuk-form-group">
         <label class="govuk-label govuk-label govuk-!-font-weight-bold" for="supporting-documentation-upload">
-          Upload supporting documentation
+          Choose supporting documents to upload
         </label>
-        <input class="govuk-file-upload" id="supporting-documentation-upload" name="supporting-documentation-upload" type="file" multiple>
+        <p class="govuk-hint" id="supporting-documentation-upload-hint">Files must be smaller than 10MB and in either a .doc, .gif, .jpg, .pdf, .png, .txt, .xls, or .zip format.</p>
```

Comment:
I can't use the NJ here as it doesn't allow us to use the `multiple` attribute on the file upload, but I've added the hint associating manually here.